scripts/run-integration-tests.sh

```diff
@@ -131,13 +131,11 @@ if [[ $(docker images -q "$IMAGE_NAME:$TEST_IMAGE_VERSION" 2> /dev/null) ]]; the
 else
     echo "CNI image $IMAGE_NAME:$TEST_IMAGE_VERSION does not exist in repository."
     START=$SECONDS
-    make docker IMAGE="$IMAGE_NAME" VERSION="$TEST_IMAGE_VERSION"
-    docker push "$IMAGE_NAME:$TEST_IMAGE_VERSION"
+    make multi-arch-cni-build-push IMAGE="$IMAGE_NAME" VERSION="$TEST_IMAGE_VERSION"
```

Comment:
Where is `TEST_IMAGE_VERSION` used in docker buildx command line?

Reply:
It is used to set Version variable in Makefile: https://github.com/aws/amazon-vpc-cni-k8s/blob/master/Makefile#L23